streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -179,31 +180,43 @@ public InternalTopicManager(final Admin adminClient, final StreamsConfig streams
      * Topics that were not able to get its description will simply not be returned
      */
     // visible for testing
-    protected Map<String, Integer> getNumPartitions(final Set<String> topics) {
-        log.debug("Trying to check if topics {} have been created with expected number of partitions.", topics);
-
-        final DescribeTopicsResult describeTopicsResult = adminClient.describeTopics(topics);
+    protected Map<String, Integer> getNumPartitions(final Set<String> topics,
+                                                    final HashSet<String> tempUnknownTopics,
+                                                    final int remainingRetries) {
```

Comment:
We could just pass in a boolean here to indicate whether there are remaining retries

Reply:
Updated.